### DISCUSSION TITLE
blog/docker-source-code-analysis-code-structure/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#317] [Announcements]
# blog/docker-source-code-analysis-code-structure/

之前陆陆续续看过一点docker的源码，都未成体系，最近在研究Docker-17.03-CE，趁此机会研究下docker的源码。

https://jimmysong.io/blog/docker-source-code-analysis-code-structure/

**Comment by mijaclan** [comment]
在看docker的源码，有好多看不懂的操作博主有相关的资料吗？
